### PR TITLE
Journal week 4

### PR DESCRIPTION
**Pull request by boshnivolo**

### CODE REVIEW

**Comment by Ricecoder** [comment]
Hilarious! I think the pigeon is definitely a sign! :+1:

**Comment by cmauban** [comment]
Nice start :+1:  Definitely sprained my ankle in the first two minutes -_-

**Comment by jeremyrist** [comment]
:+1: Maybe foul ball will come flying up there soon? haha

File: 1-month-down.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Wow, we've already finished up our 4th week here at the Iron Yard! Man, am I feeling the pressure. 
+
+With a third of our time here already past us, the incessant reminder of how short 3 months really is has been plaguing me. The fear that I'm going to find myself in week 12 just as clueless as I am now has lead me to seek a reprieve at the roof of the parking garage every day. Up there I seek a sort of sign from the downtown Durham cityscape that I can not only survive the coming weeks, but conquer them. So far I've only received a very nonchalant pigeon landing beside me and a discarded paper bag from Moe's as an answer. I'm not sure what they mean, but I took them as positive responses from the universe.
```
Comment (kellymurray) [inline 1-month-down.md:3]: very positive omens.

**Pull request merged**